packages/nextjs/src/server/clerkMiddleware.ts

```diff
@@ -62,7 +69,21 @@ interface ClerkMiddleware {
 
 export const clerkMiddleware: ClerkMiddleware = (...args: unknown[]): any => {
   const [request, event] = parseRequestAndEvent(args);
-  const [handler, options] = parseHandlerAndOptions(args);
+  const [handler, params] = parseHandlerAndOptions(args);
+  const publishableKey = assertKey(params.publishableKey || PUBLISHABLE_KEY, () =>
+    errorThrower.throwMissingPublishableKeyError(),
+  );
+  const secretKey = assertKey(params.secretKey || SECRET_KEY, () => errorThrower.throwMissingSecretKeyError());
```

Comment:
🙃 
```suggestion
  const secretKey = assertKey(params.secretKey || SECRET_KEY, errorThrower.throwMissingSecretKeyError);
```

(same for the rest)

Reply:
This results in an eslint error.